pkg/apis/service/validation/validation_test.go

```diff
@@ -219,6 +219,35 @@ var _ = Describe("Validation", func() {
 				"Field": Equal("issuers[0].skipDNSChallengeValidation"),
 			})),
 		)),
+		Entry("Valid configuration with precheck nameservers", service.CertConfig{
+			Issuers: []service.IssuerConfig{
+				{
+					Name:                "issuer",
+					Server:              "https://acme-v02.api.letsencrypt.org/directory",
+					Email:               "john@example.com",
+					PrecheckNameservers: []string{"8.8.8.8", "8.8.4.4:53", "some.special.dns.server.:53", "2001:db8::1428:57ab", "[2001:db8::1428:57ab]:53"},
+				},
+			},
+		}, BeEmpty()),
+		Entry("Invalid configuration with invalid precheck nameservers", service.CertConfig{
+			Issuers: []service.IssuerConfig{
+				{
+					Name:                "issuer",
+					Server:              "https://acme-v02.api.letsencrypt.org/directory",
+					Email:               "john@example.com",
+					PrecheckNameservers: []string{"8.8.8", "8.8.4.4:100053", "some.special!.dns.server.:53", "2001:db8::1428:57ab"},
```

Comment:
Shouldn't there be an error for `8.8.8` as well?

Reply:
If no port is specified, default DNS port `53` is assumed.

Reply:
Fixed validation for edge case `8.8.8`, as it is neither a valid IP address nor a valid domain name.